Cyberark fix param encryption (#11858)
demisto/content: Packs/cyberark_AIM/ReleaseNotes/1_0_7.md

#### Integrations
##### CyberArk AIM v2

> - Fixed an issue where the parameters **Certificate File as Text** should not be encrypted.

Reviewer: ```suggestion
- Fixed an issue where the parameters **Certificate File as Text** was unnecessarily encrypted.
```

Author: done
